plugin/storage/es/spanstore/service_operation_test.go

```diff
@@ -31,9 +32,17 @@ func TestWriteService(t *testing.T) {
 		indexService := &mocks.IndexService{}
 
 		indexName := "jaeger-1995-04-21"
+		service := Service{
+			ServiceName:   "service",
+			OperationName: "operation",
+		}
+
+		serviceHash, err := model.HashCode(service)
```

Comment:
this introduces unnecessary public method to Service type. Instead I would simply add something like this:
```
func (s Service) hashCode() string {
    h := fnv.New64a()
    h.Write(s.ServiceName)
    h.Write(s. OperationName)
    return fmt.Sprintf("%x", h.Sum64())
}
```
```